test/parallel/test-dotenv-node-options.js

```diff
@@ -59,11 +60,13 @@ describe('.env supports NODE_OPTIONS', () => {
 
   it('should update UV_THREADPOOL_SIZE', async () => {
     const code = `
-      require('assert').strictEqual(process.env.UV_THREADPOOL_SIZE, '5')
+      const { test } = require('../node-api/test_uv_threadpool_size/build/${common.buildType}/test_uv_threadpool_size');
+      const size = parseInt(process.env.UV_THREADPOOL_SIZE || 4, 10);
```

Comment:
Why is the fallback needed?

Reply:
I'm not sure. I used the tests from https://github.com/nodejs/node/pull/49165.